README.md

```diff
@@ -88,6 +92,9 @@ dokkaHtml.configure {
 }
 ```
 
+**NOTE:** replace `Html` in the above code with whatever [output format](#output_formats) you need. 
```

Comment:
I feel like this was explained in the `using the gradle plugin` and `applying plugins` sections:

> The plugin adds dokkaHtml, dokkaJavadoc, dokkaGfm and dokkaJekyll tasks to the project.

> Dokka plugin creates Gradle configuration for each output format in the form of dokka${format}Plugin:

I think it's a good idea to keep `README.md` short, do you really think it's worth to have it here?

This can definitely be added to [android plugin](https://kotlin.github.io/dokka/1.6.10/user_guide/android-plugin/android-plugin/) page, won't hurt